task/source-build/0.1/source-build.yaml

```diff
@@ -80,6 +80,7 @@ spec:
         registry_allowlist="
         registry.access.redhat.com
         registry.redhat.io
+        quay.io
```

Comment:
Does it make sense to limit it to specific org(s) ?

Reply:
I think users can technically configure the build pipeline to push to any Quay organization they want, no?

Reply:
I opened https://github.com/redhat-appstudio/build-tasks-dockerfiles/pull/70, which may be helpful for configuring this list in a more flexible way.